exp/maps/maps.go

```diff
@@ -0,0 +1,15 @@
+package maps
+
+import (
+	"golang.org/x/exp/constraints"
+	"golang.org/x/exp/maps"
+	"golang.org/x/exp/slices"
```

Comment:
```suggestion
	"slices"
```

Reply:
will require a highger go version... which is fine I think